test/lib/ClaimHashLib.t.sol

```diff
@@ -58,47 +58,47 @@ contract ClaimHashLibTester {
     }
 
     function callToMessageHashes(Claim calldata claim) external view returns (bytes32 claimHash, bytes32 typehash) {
-        return ClaimHashLib.toMessageHashes(claim);
+        return ClaimHashLib.toClaimHashAndTypehash(claim);
     }
 
     function callToMessageHashes(BatchClaim calldata claim)
         external
         view
         returns (bytes32 claimHash, bytes32 typehash)
     {
-        return ClaimHashLib.toMessageHashes(claim);
+        return ClaimHashLib.toClaimHashAndTypehash(claim);
     }
 
     function callToMessageHashes(MultichainClaim calldata claim)
         external
         view
         returns (bytes32 claimHash, bytes32 typehash)
     {
-        return ClaimHashLib.toMessageHashes(claim);
+        return ClaimHashLib.toClaimHashAndTypehash(claim);
     }
 
     function callToMessageHashes(BatchMultichainClaim calldata claim)
         external
         view
         returns (bytes32 claimHash, bytes32 typehash)
     {
-        return ClaimHashLib.toMessageHashes(claim);
+        return ClaimHashLib.toClaimHashAndTypehash(claim);
     }
 
     function callToMessageHashes(ExogenousMultichainClaim calldata claim)
         external
         view
         returns (bytes32 claimHash, bytes32 typehash)
     {
-        return ClaimHashLib.toMessageHashes(claim);
+        return ClaimHashLib.toClaimHashAndTypehash(claim);
     }
 
     function callToMessageHashes(ExogenousBatchMultichainClaim calldata claim)
         external
         view
         returns (bytes32 claimHash, bytes32 typehash)
     {
-        return ClaimHashLib.toMessageHashes(claim);
+        return ClaimHashLib.toClaimHashAndTypehash(claim);
```

Comment:
should update the test function name to `callToClaimHashAndTypehash` to reflect this change